### ISSUE TITLE
[Feature request] Fallback address

### ISSUE THREAD

**Issue by pierreis** [anonaddy/anonaddy#441]
**Problem**
Currently, if a message delivery fails, it is not possible to recover it. This tends to be problematic when transferring messages to iCloud addresses, that seems to reject emails quite easily due to the sensitive spam filter, presumably.

**Solution**
The ability to set a fallback address. If delivery fails to the primary address, delivery is attempted to the fallback address before being marked as failed.

**Comment by willbrowningme** [comment]
Thanks, that's a good idea and definitely something I would like to add in the future.

For now though I recently added the ability to choose to temporarily store any failed deliveries so that they can be downloaded and viewed. You can enable this in your account settings.

**Comment by ririko5834** [comment]
good idea. I think that there could be an option to set catch-all email address for custom domains and usernames that you can set custom receipt where emails sent to unexisting, deleted or forgotten aliases will be resend if it is enabled.

**Comment by willbrowningme** [comment]
I have just added the ability to resend failed deliveries and you can choose a different recipient(s) to resend to.

Go to the [failed deliveries](https://app.addy.io/failed-deliveries) page and you will see "Resend" next to "Download" and "Delete".

Currently only available on "Forward" email types.

**Comment by anondev-sudo** [comment]
Where can I download this update to my selfhosted version? I don't see new release

**Comment by willbrowningme** [comment]
It's not available on self-hosted instances yet. It will be in the next release.

**Comment by anondev-sudo** [comment]
It still wasn't added in the latest release :/
